style_guides/accessibility_guide.md

```diff
@@ -0,0 +1,166 @@
+# Accessibility (A11Y) Guide
+
+This document provides some technical guidelines how to prevent several common
+accessibility issues.
+
+## Interactive elements
+
+### Use `<button>` and `<a>`
+
+**TL;DR** *Use `<button>` and `<a>` (with `hred`) instead of click listeners on other elements
+and style it whatever way you need.*
+
+If you want to make an element clickable, use a `<button>` or `<a>` element for it.
+Use a *button* whenever it causes an action on the current page, and an *a* if it
```

Comment:
I think this can be slightly improved by consistently referencing these elements using codeblocks, e.g. `<button>` and `<a href>` instead of *button* and *a*.